tests/pingpong-shared/host/host.cpp

```diff
@@ -24,7 +24,7 @@ void Pong(const char* in, char* out)
         {
             got_pong = true;
         }
-        strcpy(out, in);
+        strcpy_s(out, 128, in);
```

Comment:
Where is this magic number 128 coming from? Consider adding a shared header for the enc & host and referencing constants from it, here and in the other files where this pattern occurs.

Reply:
Found the number in the declaration for the static buffer size when I was trying to determine the size for strcpy_s(). I will define a constant first.